include/zephyr/drivers/i2c_emul.h

```diff
@@ -20,23 +20,23 @@
  * @{
  */
 
-#include <zephyr/types.h>
-#include <zephyr/device.h>
-#include <zephyr/drivers/emul.h>
-
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+#include <zephyr/device.h>
+#include <zephyr/drivers/emul.h>
+#include <zephyr/types.h>
```

Comment:
Same here, this works but can't be the right solution

Reply:
Done